changelog/2025.2.0.rst

```diff
@@ -57,12 +57,13 @@ Custom components were deprecated over a year ago and, as promised, support for
 encourage all developers to migrate their custom components to :doc:`/components/external_components`. Please see
 :ref:`a_note_about_custom_components` for a detailed explanation and next steps.
 
-Removal of ``armv7`` Docker Support
------------------------------------
+Removal of ``armv7`` Docker (and HA add-on) Support
+---------------------------------------------------
 
-ESPHome no longer supports ``armv7`` hardware. This is due to both waning support as it relates to tooling and
+ESPHome no longer supports ``armv7`` (32-bit ARM) hardware. This is due to both waning support as it relates to tooling and
 performance reasons. We strongly recommend moving to a more modern architecture, especially if you're using the ESPHome
 Device Builder to build/compile firmware for your devices.
+`More detailed explanation. <https://github.com/esphome/issues/issues/6770#issuecomment-2672644628>`__
```

Comment:
It shouldn't be needed to link issues. If there is missing context here, we should add it. Driving users into technical issues isn't something we should aim for.

Reply:
Do you want me to copy that section into here?  I was trying to avoid bloating the changelog needlessly.

Reply:
That is not what I'm suggesting. I'm suggesting that if the context is incomplete, we need to complete it.